modules/salt/pages/formula-monitoring.adoc

```diff
@@ -0,0 +1,86 @@
+. In the [guimenu]``Dashboards`` section, check the dashboards you want to use:
+* [guimenu]``Uyuni server dashboard``
+* [guimenu]``Uyuni clients dashboard``
+* [guimenu]``PostgreSQL dashboard``
+* [guimenu]``Apache HTTPD dashboard``
+* [guimenu]``Kubernetes cluster dashboard``
+* [guimenu]``Kubernetes etcd dashboard``
+* [guimenu]``Kubernetes namespaces dashboard``
+. Click btn:[Save Formula] to save your configuration.
+
+
+
+.Procedure: Configuring the Prometheus Formula
+. Navigate to the menu:Formulas[Prometheus] tab, and set these parameters in the [guimenu]``Prometheus`` section:
+* Check the [guimenu]``Enabled`` box to enable Prometheus monitoring.
+* In the [guimenu]``Scrape interval`` field, type the frequency of data scraping, in seconds.
+For example, ``15`` will scrape data every fifteen seconds.
+* In the [guimenu]``Evaluation interval`` field, type the FIXME, in seconds.
+For example, ``15`` will FIXME every fifteen seconds.
+. In the [guimenu]``Uyuni Server`` section, set these parameters:
+* Check the [guimenu]``Enabled`` box to enable monitoring on this server.
+* Check the [guimenu]``Autodiscover clients`` box to enable Prometheus to automatically find and monitor new clients when they are added to the server.
+* In the [guimenu]``Username`` field, type the user name of the Prometheus account on the server.
+* In the [guimenu]``Password`` field, type the password of the Prometheus account on the server.
+. In the [guimenu]``Alerting`` section, set these parameters:
+* Check the [guimenu]``Enable local Alertmanager service`` box to enable the alert manager service.
+* Check the [guimenu]``Use local Alertmanager`` box to use the local alert manager service.
+. For each alert manager you want to use, in the menu:Alerting[Alertmanagers] section, click btn:[+], and set these parameters:
+* In the [guimenu]``IP Address:Port`` field, type the location of the alert manager target, including the port number.
+//For example, ``FIXME``.
+. To use a rule file, in the menu:Alerting[Rule Files] section, click btn:[+], and set these parameters:
+* In the [guimenu]``Rule Files`` field, type the location of the rule file you want to use.
+//For example, ``FIXME``.
+. To use a scrape configuration, in the [guimenu]``User defined scrape configurations`` section, click btn:[+], and set these parameters:
+* In the [guimenu]``Job name`` field, type a name for your configuration.
+* In the [guimenu]``Files`` field, type the location of the configuration file you want to use.
+. Click btn:[Save Formula] to save your configuration.
+
+
+
+
+.Procedure: Configuring the Prometheus Exporters Formula
+. Navigate to the menu:Formulas[Prometheus Exporters] tab, and set these parameters in the [guimenu]``Node Exporter`` section:
+* Check the [guimenu]``Enabled`` box to enable the node exporter.
+* In the [guimenu]``Arguments`` field, type any customized arguments for this exporter.
+For example, ``--web.listen-address=":9100"``.
```

Comment:
is the colon actually required here?  if yes, I'd rate it as an implementation issue.

Reply:
Dunno, I copied it from the UI.